common/lib/capa/capa/templates/optioninput.html

```diff
@@ -1,11 +1,19 @@
 <% doinline = "inline" if inline else "" %>
 
 <form class="inputtype option-input ${doinline}">
+  <label class="problem-group-label" for="input_${id}">
+    ${response_data['label']}
+  </label>
+
+  % for desc_id, desc_text in response_data['descriptions'].iteritems():
+    <p id="${desc_id}}">${desc_text}</p>
```

Comment:
@ehteshamkafeel You need to add `class="question-description"` on `<p>` tag. This will used in unit tests to verify descriptions info.